stglib/iq.py

```diff
@@ -254,13 +305,41 @@ def cdf_to_nc(cdf_filename):
     # add lat/lon coordinates
     ds = utils.ds_add_lat_lon(ds)
 
-    ds = rename_vars(ds)
+    # should function this
+    for var in ds.data_vars:
+        ds = qaqc.trim_min(ds, var)
+        ds = qaqc.trim_max(ds, var)
+        ds = qaqc.trim_min_diff(ds, var)
+        ds = qaqc.trim_max_diff(ds, var)
+        ds = qaqc.trim_med_diff(ds, var)
+        ds = qaqc.trim_med_diff_pct(ds, var)
+        ds = qaqc.trim_bad_ens(ds, var)
+        ds = qaqc.trim_maxabs_diff_2d(ds, var)
+        ds = qaqc.trim_fliers(ds, var)
 
-    ds = ds_add_attrs(ds)
+    # after check for masking vars by other vars
+    for var in ds.data_vars:
+        ds = qaqc.trim_mask(ds, var)
 
-    ds = utils.no_p_create_depth(ds)
+    ds = utils.create_z(ds)  # added 7/31/2023
 
-    ds = ds.drop(["SampleNumber", "SampleTime"])
+    ds = utils.add_standard_names(ds)
+
+    ds = ds_add_attrs(ds)
+
+    # ds = utils.no_p_create_depth(ds) #commented out 7/31/23
+
+    ds = ds.drop(
```

Comment:
Are we guaranteed to have these variables always or do we need to check for their existence?